scripts/get-fonts.sh

```diff
@@ -17,99 +17,117 @@ download() {
+NotoSansBalinese;balinese
+NotoSansBamum;bamum
+NotoSansBengaliUI;bengali
+NotoSansCanadianAboriginal;canadian-aboriginal
+NotoSansCham;cham
+NotoSansCherokee;cherokee
+NotoSansDevanagariUI;devanagari
+NotoSansEthiopic;ethiopic
+NotoSansGeorgian;georgian
+NotoSansGujaratiUI;gujarati
+NotoSansGurmukhiUI;gurmukhi
+NotoSansHebrew;hebrew
+NotoSansJavanese;javanese
+NotoSansKannadaUI;kannada
+NotoSansKayahLi;kayah-li
+NotoSansKhmerUI;khmer
+NotoSansLaoUI;lao
+NotoSansLisu;lisu
+NotoSansMalayalamUI;malayalam
+NotoSansMyanmarUI;myanmar
+NotoSansOlChiki;ol-chiki
+NotoSansOriya;oriya
+NotoSansSinhalaUI;sinhala
+NotoSansSundanese;sundanese
+NotoSansSymbols;symbols
+NotoSansTaiTham;tai-tham
+NotoSansTamilUI;tamil
+NotoSansTeluguUI;telugu
+NotoSansThaana;thaana
+NotoSansThaiUI;thai
+NotoSerifTibetan;tibetan"
 
 # Fonts with regular and black, but no bold
-REGULAR_BLACK="NotoSansSyriac"
+REGULAR_BLACK="NotoSansSyriac;syriac"
 
 # Fonts only available in regular
-REGULAR="NotoSansBatak
-NotoSansBuginese
-NotoSansBuhid
-NotoSansChakma
-NotoSansCoptic
-NotoSansHanunoo
-NotoSansLepcha
-NotoSansLimbu
-NotoSansMandaic
-NotoSansMongolian
-NotoSansNewTaiLue
-NotoSansNKo
-NotoSansOsage
-NotoSansOsmanya
-NotoSansSamaritan
-NotoSansSaurashtra
-NotoSansShavian
-NotoSansSymbols2
-NotoSansTagalog
-NotoSansTagbanwa
-NotoSansTaiLe
-NotoSansTaiViet
-NotoSansTifinagh
-NotoSansVai
-NotoSansYi"
+REGULAR="NotoSansBatak;batak
+NotoSansBuginese;buginese
+NotoSansBuhid;buhid
+NotoSansChakma;chakma
+NotoSansCoptic;coptic
+NotoSansHanunoo;hanunoo
+NotoSansLepcha;lepcha
+NotoSansLimbu;limbu
+NotoSansMandaic;mandaic
+NotoSansMongolian;mongolian
+NotoSansNewTaiLue;new-tai-lue
+NotoSansNKo;nko
+NotoSansOsage;osage
+NotoSansOsmanya;osmanya
+NotoSansSamaritan;samaritan
+NotoSansSaurashtra;saurashtra
+NotoSansShavian;shavian
+NotoSansSymbols2;symbols
+NotoSansTagalog;tagalog
+NotoSansTagbanwa;tagbanwa
+NotoSansTaiLe;tai-le
+NotoSansTaiViet;tai-viet
+NotoSansTifinagh;tifinagh
+NotoSansVai;vai
+NotoSansYi;yi"
 
 # Download the fonts in the lists above
 
 for font in $REGULAR_BOLD_ITALIC; do
   regular="$font-Regular.ttf"
   bold="$font-Bold.ttf"
   italic="$font-Italic.ttf"
-  download "${FONTDIR}/${regular}" "https://github.com/notofonts/noto-fonts/raw/main/hinted/ttf/${font}/${regular}"
-  download "${FONTDIR}/${bold}" "https://github.com/notofonts/noto-fonts/raw/main/hinted/ttf/${font}/${bold}"
-  download "${FONTDIR}/${italic}" "https://github.com/notofonts/noto-fonts/raw/main/hinted/ttf/${font}/${italic}"
+  download "${FONTDIR}/${regular}" "https://notofonts.github.io/latin-greek-cyrillic/fonts/${font}/hinted/ttf/${regular}"
+  download "${FONTDIR}/${bold}" "https://notofonts.github.io/latin-greek-cyrillic/fonts/${font}/hinted/ttf/${bold}"
+  download "${FONTDIR}/${italic}" "https://notofonts.github.io/latin-greek-cyrillic/fonts/${font}/hinted/ttf/${italic}"
 done
 
-for font in $REGULAR_BOLD; do
+for fontDir in $REGULAR_BOLD; do
+  IFS=";" read -r -a arr <<< "${fontDir}"
```

Comment:
This does not work with POSIX `sh` and requires Bash: https://stackoverflow.com/questions/2462317/bash-syntax-error-redirection-unexpected

It makes the Docker setup fail.

Changing the first line to `#!/usr/bin/env bash` fixes it.

Reply:
✔️  I'm not familiar with different shells, but made the change

Reply:
Changed to Python script